docs/path-resolution.rst

```diff
@@ -464,7 +464,8 @@ Here are the detailed rules governing the behaviour of remappings:
 
 #. **Prefix cannot be empty but context and target are optional.**
 
-   If ``target`` is omitted, it defaults to the value of the ``prefix``.
+   - If ``target`` is omitted, ``prefix`` is simply removed from import paths.
```

Comment:
Does `omitted` mean that it is empty or that also the `=` is missing?

Reply:
Empty with `=` still present. I do not mention it explicitly because I don't think any user will really think about just `prefix` alone as a remapping unless we specifically mention it.

Reply:
Would "If target is the empty string" be a better description in this case?

Reply:
Yeah, that sounds unambiguous to me.